webpack/assets/javascripts/react_app/components/PasswordStrength/PasswordStrength.js

```diff
@@ -12,8 +12,9 @@ export default class PasswordStrength extends React.Component {
       updatePassword,
       updatePasswordConfirmation,
       doesPasswordsMatch,
+      passwordPresent,
       data: {
-        className, id, name, verify, error, userInputIds,
+        className, id, name, verify, error, userInputIds, required,
```

Comment:
Can you please add the `passwordPresent` and the `required` fields to the propTypes?